release_files/Dockerfile.ubuntu

```diff
@@ -80,64 +81,60 @@ SHELL ["/bin/bash", "-c"]
 WORKDIR /
 
 COPY drivers /drivers
-
-RUN	set -e ; \
-	set -x ; \
-        apt update -y ; \
-        apt install -y curl libpugixml1v5 libtbb2 && \
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+RUN	apt-get update -y ; \
+    apt-get install -y curl libpugixml1v5 libtbb2 --no-install-recommends && \
 	if [ "$GPU" == "1" ] ; then \
-                apt-get update && apt-get  install -y libnuma1 ocl-icd-libopencl1 && rm -rf /var/lib/apt/lists/* && \
+                apt-get update && apt-get  install -y libnuma1 ocl-icd-libopencl1 --no-install-recommends && rm -rf /var/lib/apt/lists/* && \
 	        case $INSTALL_DRIVER_VERSION in \
                 "20.35.17767") \
-                        mkdir /tmp/gpu_deps && cd /tmp/gpu_deps ; \
+                        mkdir /tmp/gpu_deps && \
                         curl -L -O https://github.com/intel/compute-runtime/releases/download/20.35.17767/intel-gmmlib_20.2.4_amd64.deb ; \
                         curl -L -O https://github.com/intel/compute-runtime/releases/download/20.35.17767/intel-igc-core_1.0.4756_amd64.deb ; \
                         curl -L -O https://github.com/intel/compute-runtime/releases/download/20.35.17767/intel-igc-opencl_1.0.4756_amd64.deb ; \
                         curl -L -O https://github.com/intel/compute-runtime/releases/download/20.35.17767/intel-opencl_20.35.17767_amd64.deb ; \
                         curl -L -O https://github.com/intel/compute-runtime/releases/download/20.35.17767/intel-ocloc_20.35.17767_amd64.deb ; \
                         curl -L -O https://github.com/intel/compute-runtime/releases/download/20.35.17767/intel-level-zero-gpu_1.0.17767_amd64.deb ; \
-                        dpkg -i intel*.deb && rm -Rf /tmp/gpu_deps ; \
+                        dpkg -i /tmp/gpu_deps/intel*.deb && rm -Rf /tmp/gpu_deps ; \
                 ;; \
                 "21.48.21782") \
-                        mkdir /tmp/gpu_deps && cd /tmp/gpu_deps ; \
```

Comment:
Are you sure this will work? Below curls are not specifying /tmp/gpu_deps catalog as output so the installation with dpkg might not work.